makeservices/makemysql

```diff
@@ -3,4 +3,7 @@ if [[ "$(hostname)" != "tsunami" && "$(hostname)" != "dev-tsunami" ]]; then
     echo -e '\033[1;31mYou must run this command on tsunami.\033[0m'
     exit 1
 fi
-sudo -u mysql /opt/share/utils/makeservices/makemysql-real
+
+PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real | tee /dev/tty | tail -n 1 | grep -Po '(?<=: )([0-9a-zA-Z]){24,}$')
+cd ~/public_html/
+wp config set DB_PASSWORD "$PASS" > /dev/null 2>&1
```

Comment:
Why are you throwing out this error? Silent failures can create a lot of confusion.

Reply:
I wanted to make sure the script didn't interfere with other scripts that use the makemysql command. Ill modify to only throw out the stdout

Reply:
Hmm, that's a good point, stripping stdout is probably OK here.